site/content/extend/getting-started/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Getting Started"
+date: "2018-04-19T12:01:23-04:00"
+section: "extend"
+---
+
+# Getting Started
+
+Mattermost can be extended by building and installing plugins, by embedding Mattermost in another application, or by forking the codebase.
+
+## Extension Methods
+
+### Plugins (Beta)
+
+Plugins are a powerful way to add new functionality and UI to Mattermost - in an easy to maintain and update way. They are easy to build and give you more power than any other integration method.
+
+[Read more about plugins.](/extend/plugins)
+
+### Embed
+
+Mattermost is a great communication platform but sometimes it's more convenient to send and receive messages from within a non-Mattermost application. With Mattermost's rich RESTful web service API and OAuth 2.0 support it's possible to build your own clients to embed Mattermost within any application of your choice.
+
+[Read more about embedding.](/extend/embed)
+
+### Fork
+
+Mattermost is open-source, which means you have the ability to fork the source code if you need to modify Mattermost in ways not otherwise supported.
+
+[Read about licensing around forking Mattermost.](https://github.com/mattermost/mattermost-server/blob/master/LICENSE.txt)
```

Comment:
completely 0/5, but we also have this FAQ that might be easier to read https://docs.mattermost.com/overview/faq.html#how-can-i-create-a-derivative-work-of-mattermost-as-my-own-commercial-solution